experimental/packages/otlp-transformer/test/metrics.test.ts

```diff
@@ -24,14 +24,17 @@ import {
 import * as assert from 'assert';
 import { createExportMetricsServiceRequest } from '../src/metrics/internal';
 import { EAggregationTemporality } from '../src/metrics/internal-types';
-import { hrTime, hrTimeToNanoseconds } from '@opentelemetry/core';
+import { getTimeOrigin, millisecondsToNanoseconds } from '@opentelemetry/core';
 import * as root from '../src/generated/root';
-import { encodeAsLongBits, encodeAsString } from '../src/common/utils';
+import { toLongBits } from '../src/common/utils';
 import { ProtobufMetricsSerializer } from '../src/metrics/protobuf';
 import { JsonMetricsSerializer } from '../src/metrics/json';
 
-const START_TIME = hrTime();
-const END_TIME = hrTime();
+const START_TIME = millisecondsToNanoseconds(
+  performance.now() + getTimeOrigin()
+);
+const END_TIME = millisecondsToNanoseconds(performance.now() + getTimeOrigin());
```

Comment:
I found multiple occurrences of `hrTime()` need to be converted to nanos. Maybe we can add a util function `nowInNanos()`.